app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :votes
+  validates :username, presence: true, uniqueness: true
+
+  def self.voted_works(user_id)
+    works = User.where(id: user_id)
+    return works
+  end
```

Comment:
This method returns a list of (one) user, not a list of works. Why not use a `has_many :through` relationship here?